src/transformers/models/time_series_transformer/configuration_time_series_transformer.py

```diff
@@ -0,0 +1,153 @@
+    for more information.
+
+
+    Args:
+        prediction_length (`int`):
+            The prediction horizon for the model.
+        context_length (`int`, *optional*, default to `None`):
+            The context length for the encoder. If  `None`, the context length will be the same as the prediction length.
+        distr_output (`string` default to `StudentT`):
+            The distribution emission head for the model.
+        loss (`string` default to `NLL`):
+            The loss function for the model with corresponding to the `distr_output` head.
+        input_size (`int` default to 1):
+            The size of the target variable which by default is 1 for univariate targets.
+        scaling (`bool` default to `True`):
+            Whether to scale the input targets.
+        freq (`str`, *optional* default to `None`):
+            The frequency of the input time series. If `None`, the `lags_seq` and `num_time_features` are set at the finest temporal resolution of 1 Second.
+        lags_seq (`list` of `int`, *optional* default to `None`):
+            The lags of the input time series. If `None`, the `freq` is used to determine the lags.
+        num_feat_dynamic_real (`int`, *optional* default to `0`):
+            The number of dynamic real valued features.
+        num_feat_static_cat (`int`, *optional* default to `0`):
+            The number of static categorical features.
+        num_feat_static_real (`int`, *optional* default to `0`):
+            The number of static real valued features.
+        cardinality (`list` of `int`, *optional* default to `None`):
+            The cardinality of the categorical features. Cannot be `None` if `num_feat_static_cat` is `> 0`.
+        embedding_dimension (`list` of `int`, *optional* default to `None`):
+            The dimension of the embedding for the categorical features. Cannot be `None` if `num_feat_static_cat` is `> 0`.
+        encoder_layers (`int`, *optional*, defaults to 2):
+            Number of encoder layers.
+        decoder_layers (`int`, *optional*, defaults to 2):
+            Number of decoder layers.
+        nhead (`int`, *optional*, defaults to 2):
+            Number of attention heads for each attention layer in the Transformer encoder and decoder.
+        ffn_dim (`int`, *optional*, defaults to 32):
+            Dimension of the "intermediate" (often named feed-forward) layer in encoder and decoder.
+        activation_function (`str` or `function`, *optional*, defaults to `"gelu"`):
+            The non-linear activation function (function or string) in the encoder and decoder. If string,
+            `"gelu"` and `"relu"` are supported.
+        dropout (`float`, *optional*, defaults to 0.1):
+            The dropout probability for all fully connected layers in the encoder, and decoder.
+
+        Example:
+
+    ```python
+    >>> from transformers import TimeSeriesTransformerModel, TimeSeriesTransformerConfig
```

Comment:
```suggestion
    >>> from transformers import TimeSeriesTransformerConfig, TimeSeriesTransformerModel
```
The CookieCutter always imports them in the wrong order..